runtime/src/main/java/org/corfudb/protocols/wireprotocol/NettyCorfuMessageEncoder.java

```diff
@@ -17,24 +18,30 @@
 
     final LongAccumulator maxValue = new LongAccumulator(Math::max, Long.MIN_VALUE);
 
+    final int maxMessageSize;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4703651)